### DISCUSSION TITLE
Row for ?

### DISCUSSION THREAD

**Discussion by joaopaulobiesek** [QuestPDF/QuestPDF#1000] [General]
I have a PDF and I need to launch a for and use row. however, when I create an int row I receive the error: (CS1503 Argument 2: it is not possible to convert from "int" to "System.Action<QuestPDF.Fluent.RowDescriptor>"). Is there any way to get around this?
![image](https://github.com/user-attachments/assets/2161d786-5a9b-4194-8c80-6fab0af55b88)

**Comment by joaopaulobiesek** [comment]
Get instead of USE "int". The use to avoid the error must be "uint"!

**Comment by DawidRobertNowak** [comment]
Hello,

Glad you were able to find the solution! We are closing this thread as resolved. Thank you for sharing your answer — it may help others with the same question.

Thank you!
